src/components/_gapminder/buttonlist/buttonlist.js

```diff
@@ -178,6 +187,19 @@
       close_buttons.on('click', function () {
         _this.closeAllDialogs();
       });
+      var pinDialog = this.element.selectAll("[data-click='pinDialog']");
+      pinDialog.on('click', function () {
+        _this.pinDialog(this);
+      });
+
+       d3.selectAll(".vzb-buttonlist-container-dialogs").on('click', function(){
+         d3.event.preventDefault();
+         d3.event.stopPropagation();
+       });
+
+      document.getElementById('vzbp-main').addEventListener('click', function(){
```

Comment:
We cant bind it to this div. It should be bound to internal vizabi div only. '#vzbp-main' is part of the external page, not Vizabi. @maximhristenko . Theres a reference to the root element in `this.root`

Can you change it to `this.root.addEventListener` @maximhristenko ? Thanks